packages/lwt/lwt.2.7.1/opam

```diff
@@ -0,0 +1,68 @@
+opam-version: "1.2"
+name: "lwt"
+version: "2.7.1"
+maintainer: [
+  "Anton Bachin <antonbachin@yahoo.com>"
+  "Mauricio Fernandez <mfp@acm.org>"
+  "Simon Cruanes <simon.cruanes.2007@m4x.org>"
+]
+authors: [
+  "J√©r√¥me Vouillon"
```

Comment:
What the...?

Thanks, I replaced the commit. The new `opam` file is encoded correctly, I believe.